h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -615,23 +610,23 @@ private void pollAndUpdateProgress(Stage stage, String name, WorkAllocations.Wor
     // pick up any stragglers:
     if (JobType.HyperparamSearch == work.type) {
       if (subJob.isCrashed()) {
-        userFeedback.info(stage, name + " failed: " + subJob.ex().toString());
+        userFeedback.warn(stage, name + " failed: " + subJob.ex().toString());
       } else if (subJob.get() == null) {
-        userFeedback.info(stage, name + " cancelled");
+        userFeedback.warn(stage, name + " cancelled");
       } else {
-        Grid grid = (Grid) subJob.get();
+        Grid<?> grid = (Grid) subJob.get();
         int gridCount = grid.getModelCount();
-        if (gridCount > gridLastCount) {
-          userFeedback.info(Stage.ModelTraining, "Built: " + gridCount + " models for search: " + name);
+        if (gridCount > lastTotalGridModelsBuilt) {
```

Comment:
@seb-h2o gridCount -> totalGridModelsBuilt

Reply:
you see? you ask to rename legacy code and now we start rechecking everything.
will rename, but can you still please approve? usage is to approve as soon as there's no major concern/unanswered question, and trust author for the rest, thx.

Reply:
@seb-h2o I don't agree with suggested attitude. Once your agreed to change legacy code then let's do it consistently.